README.md

```diff
@@ -143,6 +143,11 @@ VARIABLE_WITH_DEFAULT: !var:default='defaultvalue' path/to/variable
 
 * `-f <path>` specify a location to a secrets.yml file, default 'secrets.yml' in current directory.
 
+* `-u, up` searches for secrets.yml going up, starting from the current working directory
```

Comment:
I think for consistency, this should be '--up' rather than 'up' without the double dashes.

Reply:
@BradleyBoutcher  Added the `--`.
@sgnn7 I am fine with adding a `.`, but note that all the options except `-f` and `D` actually lack a terminating period. I can add all of them, let me know.

Reply:
@marco-m 👍 feel free to choose if you want the `.` included or not but just make sure that all items are consistent with the pattern

Reply:
Added `.` to all the options.